rollup.config.js

```diff
@@ -1 +1,13 @@
-module.exports = require('skatejs-build/rollup.config');
+const config = require('skatejs-build/rollup.config');
+const pkg = require('skatejs-build/package');
+
+const deps = Object.keys(pkg.dependencies || []);
+config.external = id => {
+  if (id === 'react' || id === 'react-dom') {
+    return true;
+  }
+
+  return deps.indexOf(id) > -1
```

Comment:
This is pretty fuzzy logic, is it really what we want? Don't we just want to avoid bundling `react` and `react-dom`?